src/components/form/Lookup/FundingAccount/FundingAccountField.tsx

```diff
@@ -0,0 +1,24 @@
+import { LookupField } from '../..';
+import { CreateFundingAccountInput } from '../../../../api';
+import { CreateFundingAccount } from '../../../../scenes/Locations/LocationForm/FundingAccount/CreateFundingAccount';
+import {
+  FundingAccountLookupItemFragment as FundingAccount,
+  useFundingAccountLookupLazyQuery,
+} from './FundingAccountField.generated';
+
+export const FundingAccountField = LookupField.createFor<
+  FundingAccount,
+  CreateFundingAccountInput
+>({
+  resource: 'FundingAccount',
+  useLookup: useFundingAccountLookupLazyQuery,
+  label: 'Funding Account',
+  placeholder: 'Search for a funding account by name',
+  CreateDialogForm: CreateFundingAccount,
+  // @ts-expect-error don't need to pass through entire initialValues
```

Comment:
Saw the same expect error tag on userfield also, but let me know if there's a better way around the error